### ISSUE TITLE
Customizable frame data / Making hover time sensible

### ISSUE THREAD

**Issue by Endaris** [panel-attack/panel-game#158]
Original discussion: https://discord.com/channels/196674532491132937/854285957347147778/935246179757850685
Originally sparked by the fact that the frame data in the original games is not monotonously increasing / decreasing with increasing levels, making a very awkward jump from 
8 to 9 (starting speed decreasing from 29 to 27 after continuously increasing with higher levels)
9 to 10 (hover time increasing from 3 to 6 after continuously decreasing with higher levels)

Technically this is very easy to "fix" but we definitely want to keep parity with the original games in the foreseeable future, making this mainly a presentation issue of how to sell this to players. Additionally it is currently undesirable to split the playerbase even further over settings that are not terrible different from each other which is why I'm tagging this with wontfix for now.

**Comment by Endaris** [comment]
General suggestion for customization that is in big parts contained in this one and its underlying problem https://discord.com/channels/196674532491132937/854285957347147778/935255236703051886

**Comment by JamesVanBoxtel** [comment]
Yeah its not that we don't want to do something here, just that its complicated / controversial and there are more important things that aren't

**Comment by Endaris** [comment]
Untagging engine and tagging graphics.
Customizable frame data is generally supported in internals and replays, there is no UI for it however.
Related:
- panel-attack/panel-game#150 
- panel-attack/panel-game#44 

Given that adding full customizability is a bit controversial in design (for vs modes) and also a daunting challenge in screen space design, implementing the two above seems more valuable as being able to play 5 color level 10 or 6 color level 6-8 already greatly softens issues with level progression.
